src/main/java/Duke.java

```diff
@@ -1,10 +1,164 @@
-public class Duke {
+import java.util.Scanner;
+
+/**
+ * This class implements the Project Duke, a Personal
+ * Assistant Chatbot that helps a user to keep track
+ * of various things. Currently, Duke can greet user;
+ * add/list user tasks, and mark them as "done".
+ *
+ * @author richwill28
+ */
+class Duke {
```

Comment:
You might want to show the access level of the class Duke?

Reply:
Thanks for pointing this out! I should declare the class as `public` instead of the `default access` to allow other packages to access it.